src/server/game/Entities/Unit/Unit.cpp

```diff
@@ -14752,3 +14752,27 @@ void Unit::Whisper(uint32 textId, Player* target, bool isBossWhisper /*= false*/
     ChatHandler::BuildChatPacket(data, isBossWhisper ? CHAT_MSG_RAID_BOSS_WHISPER : CHAT_MSG_MONSTER_WHISPER, LANG_UNIVERSAL, this, target, bct->GetText(locale, getGender()), 0, "", locale);
     target->SendDirectMessage(&data);
 }
+
+float Unit::GetCollisionHeight() const
+{
+    float scaleMod = GetObjectScale(); // 99% sure about this
+
+    if (IsMounted())
```

Comment:
Are you sure this returns the correct value during `Mount`/`Dismount`?

Reply:
Yes, but I noticed it also returns true when in vehicle. When that's the case, it will just not pass the second if and go to the dismounting case.

Reply:
Alright, good enough for me :+1: